<html class="no-js" lang="en">
<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>USTC-Software | Welcome</title>

    <meta name="description" content="iGEM">
    <meta name="author" content="Taixing Lin">
    <meta name="copyright" content="USTC-Software, inc. Copyright (c) 2014">
 
    <link href="http://2014.igem.org/Team:USTC-Software/fixed-igemHead.css?action=raw&ctype=text/css" rel="stylesheet" type="text/css" />
    <!--
<link href='http://fonts.googleapis.com/css?family=Roboto+Condensed' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Noto+Sans' rel='stylesheet' type='text/css'>
    
    -->
    <link href='http://fonts.googleapis.com/css?family=Lobster' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" href="http://2014.igem.org/Team:USTC-Software/reset.css?action=raw&ctype=text/css"> <!-- CSS reset -->
    
    
     <link rel="stylesheet" href="http://2014.igem.org/Team:USTC-Software/foundation.css?action=raw&ctype=text/css" />
     <link rel="stylesheet" href="http://2014.igem.org/Team:USTC-Software/animate.css?action=raw&ctype=text/css">
    <link rel="stylesheet" href="http://2014.igem.org/Team:USTC-Software/main-new.css?action=raw&ctype=text/css" />
    
    <link rel="stylesheet" href="http://2014.igem.org/Team:USTC-Software/style.css?action=raw&ctype=text/css"> <!-- Resource style -->
     
    <script src="http://2014.igem.org/Team:USTC-Software/modernizr.js?action=raw&ctype=text/javascript"></script>
   
    <!--[if lt IE 9]>
    <link href="http://huodong.ustc.edu.cn/Web/igem2014-website/css/ie.css" media="screen" rel="stylesheet" />
    <![endif]-->

    <!-- IE Fix for HTML5 Tags -->
    <!--[if lt IE 9]>
    <script src="http://huodong.ustc.edu.cn/Web/igem2014-website/js/html5.js"></script>
    <![endif]-->
    

</head>
  <body class="off-canvas hide-extras wiki" style="">

<!--
<aside class="left-off-canvas-menu">
  <ul class="off-canvas-list">
    <li><label class="first">USTC-Software</label></li>
    <li><a href="index.html">Home</a></li>
  </ul>

  <hr>

  <ul class="off-canvas-list">
    <li><label class="first">About</label></li>
    <li><a href="learn/features.html">Overall</a></li>
    <li><a href="learn/faq.html">Software</a></li>
    <li><a href="learn/faq.html">Method</a></li>
    <li><a href="learn/faq.html">Examples</a></li>
    <li><a href="learn/faq.html">Future Work</a></li>
  </ul>

  <hr>

</aside>
-->
<!-- -------------------------------------->

 <!--
<nav class="tab-bar show-for-small">
  <a class="left-off-canvas-toggle menu-icon" style="line-height: 1.0625rem;">
     
    <span>USTC_Software</span>
   
  </a>
</nav>
-->
<nav class="top-bar" data-topbar role="navigation">
  <ul class="title-area">
    <li class="name" >
      <h1><a href="/Team:USTC-Software" style="font-family: 'Lobster', cursive;">USTC-Software</a></h1>
    </li>
    <li class="toggle-topbar menu-icon"><a href="#"></a></li>
  </ul>
<!-- -------------------------------------->

  <section class="top-bar-section">
    <ul class="right">

      <li class="divider"></li>
      <li class="has-dropdown">
        <a href="/Team:USTC-Software/biopano.php" class="">BioPano</a>
        <ul class="dropdown">
            <li><a href="/Team:USTC-Software/biopano.php#overview">Overview</a></li>
            <li><a href="/Team:USTC-Software/biopano.php#motiation">Motivation</a></li>
            <li><a href="/Team:USTC-Software/biopano.php#features">Features</a></li>
            <li><a href="/Team:USTC-Software/biopano.php#demos">Demos</a></li>
            <li><a href="/Team:USTC-Software/biopano.php#why">Why choose BioPano?</a></li>
            <li><a href="/Team:USTC-Software/biopano.php#futurework">Future work</a></li>
        </ul>
      </li>
      <li class="divider"></li>
      <li class="has-dropdown">
        <a href="/Team:USTC-Software/develop.php" class="">Develop</a>
         <ul class="dropdown">
            <li><a href="/Team:USTC-Software/develop.php#framework">Framework</a></li>
            <li><a href="/Team:USTC-Software/develop.php#interface">Interface</a></li>
            <li><a href="/Team:USTC-Software/develop.php#cloud">Cloud</a></li>
            <li><a href="/Team:USTC-Software/develop.php#algorithm">Algorithm</a></li>
        </ul>
      </li>
      <li class="divider"></li>
      <li class="has-dropdown">
        <a href="/Team:USTC-Software/requirements.php" class="">Requirements</a>
         <ul class="dropdown">
            <li><a href="/Team:USTC-Software/requirements.php#medals">Medals</a></li>
            <li><a href="/Team:USTC-Software/requirements.php#safety">Safety</a></li>
            <li><a href="/Team:USTC-Software/requirements.php#collaboration">Collaboration</a></li>
        </ul>
       
      </li>
      <li class="divider"></li>
      <li class="has-dropdown">
        <a href="/Team:USTC-Software/human_practice.php" class="">Human Practice</a>
        <ul class="dropdown">
            <li><a href="/Team:USTC-Software/human_practice.php#management">Management</a></li>
            <li><a href="/Team:USTC-Software/human_practice.php#biology_cooperation">Biology Cooperation</a></li>
            <li><a href="/Team:USTC-Software/human_practice.php#igem_meetups">iGEM Meetups</a></li>
        </ul>
      </li>
      <li class="divider"></li>
     
      <li class="has-dropdown">
        <a href="/Team:USTC-Software/team.php" class="">Team</a>
        <ul class="dropdown">
            <li><a href="/Team:USTC-Software/team.php#team_members">Team Members</a></li>
            <li><a href="/Team:USTC-Software/team.php#I_A">Instructors & Advisors</a></li>
            <li><a href="/Team:USTC-Software/team.php#Additional_thanks">Additional Thanks</a></li>
        </ul>
      </li>
      <li class="divider"></li>
      <li class="has-dropdown">
        <a href="/Team:USTC-Software/acknowledge.php" class="">Acknowledge</a>
        <ul class="dropdown">
            <li><a href="/Team:USTC-Software/acknowledge.php#dbank">Deutsche Bank</a></li>
            <li><a href="/Team:USTC-Software/acknowledge.php#ustcif">USTCIF</a></li>
            <li><a href="/Team:USTC-Software/acknowledge.php#utao">USTC Teaching Affair Office</a></li>
            <li><a href="/Team:USTC-Software/acknowledge.php#regulondb">RegulonDB</a></li>
            <li><a href="/Team:USTC-Software/acknowledge.php#kegg">Kegg</a></li>
            <li><a href="/Team:USTC-Software/acknowledge.php#ncbi">NCBI</a></li>
        </ul>
      </li>
      <ul>
  </section>

</nav>
<div class="main-content">
     <div class="row large-12 columns large-centered wiki-head">
        <p style="font-size:64px;height:100%;line-height:100%;color:rgb(144,144,144)">Meet Biopano</p>
        
      </div>
      <div class="row">
        <div class="large-12 columns large-centered">

        
      <h4 id="overview">Overview</h4>
      <div class="underline-small"></div>
        <p>Designed specifically for biological research, BioPano is a software platform targeted for visualisation of biological relationships as well as cooperative net-building. You can set up your own network of biological relationship by adding information about genetic regulation and metabolic system to the network we already provided for you. Also, the software allows you to explore the links among the substances by just several clicks, and you can even look into the details of the interaction between the exogenous gene and the host. What is even more exciting is that our software will end your loneliness and frustration in research, since it is providing you with cloud cooperating system so that you can view massive amounts of resources in databases, together with build up groups to share information and set up mutual network with others.
        </p>
      
      <h4 id="motivation">Motivation</h4>
      <div class="underline-small"></div>
        <p>
        In recent years, people has made a lot of remarkable achievements 
in synthetic biology. However, scientists are often faced with 
many unpredictable problems when carrying out experiments. 
Each kind of biological substance is inextricably linked to the 
biological environment to which it belongs, so we have to consider 
a question: Can we attain the desired goal, if we simply follow the traditional ideas of synthetic biology and avoid the problem merely by engineering?
          </p>
          <p>
            A synthetic biologist, Chris Voigt, has successfully transferred a
Toggle Switch, which usually works in the Bacillus 
subtilis, into E. coli. Unfortunately, it did not function properly. In 
fact, in experiments, due to the unexpected interactions between 
BioBrick and the host, it is very common that the designed path do 
not work as anticipated.
          </p>
          <p>
            In fact, the life system is so complex that there are many sorts of 
biological substance to support it with complex interations in it. 
In order to understand a phenomenon, we keep using new and 
sophisticated methods to refine our research and to explore every 
detail of life. However, if we widen our perspective, by looking at 
our research objectives from a more comprehensive perspective, 
we may have some unexpected discoveries.
          </p>
    
      <h4 id="features">Features</h4>
      <div class="underline-small">
      </div>
        <p>Our software has many marvelous features, with which your biology research will become extremely easy and joyful.</p>
        <ul>
       
              <h5>1. Expansion of single node</h5>
                <div class="row">
                    <div class="large-6 columns"><p>In Biopano, every biology part of E.coli K-12 will be displayed as a node of different types and regulation relationship 
will be  displayed as arc of different types. They are displayed in the biological network in a visualizational way. 
You just need to select a node and click “Expand”, and the nodes associated with it will be “expanded”. 
Nodes are connected to each other by all kinds of relationship, so users can see how biological parts are connected clearly, 
such as LacI Operon. </p></div>
                    <div class="large-6 columns"><img src="http://2014.igem.org/wiki/images/4/46/2014ustc-Feature-a.png"></div>
                </div>

                <div class="row">
                    <div class="large-6 columns"><img src="http://2014.igem.org/wiki/images/9/9e/2014ustc-Feature-b.png"></div>
                    <div class="large-6 columns"><p>However, since there are so many biological parts connected to each other in various convoluted, 
the expansion of all nodes will give rise to extremely involved nets, which will fail to convey useful information to the users, 
even interfere their thoughts. So you can choose to expand the nodes in a way you see fit. Take Transcription factor CRP as an example. </p>
<p>Biopano displays the network dynamically, which makes you able to tease out the relationship among thousands of materials of 
E.coli K-12, satisfies your curiosity and helps you discover unknown biological field.</p></div>
                </div>
                <div class="underline-tiny"></div>
       

       
                <h5>2. Link Finder</h5>
                   <div class="row">
                    <div class="large-6 columns"><p>You can input two nodes that seem irrelevant, such as gene A and Transcription FactorB, and the software will 
search the route connecting the two nodes for you. Since some routes are too sinuous for further analysis or does not make sense, 
you can set a specific number k, Biopano will search and show the k shortest routes for you.</p></div>
                    <div class="large-6 columns"><img src="http://2014.igem.org/wiki/images/7/79/2014ustc-Feature-c.png"></div>
                    </div>
                    <div class="underline-tiny"></div>
                
        
                <h5>3. BLAST</h5>
                <div class="row">
                    <div class="large-6 columns"><img src="http://2014.igem.org/wiki/images/7/71/2014ustc-Feature-e.png"></div>
                    <div class="large-6 columns"><p>By BLAST method, Biopano finds E.coli K-12 gene highly similar to the injected sequence. 
These genes are regulated by other parts in E.coli K-12, so can provide information of the host environment’s impact on injected 
exogenous sequence. Biopano also offers BioBrick helper, enabling you search all kinds of BioBricks on iGEM’s official website. 
By BLAST analysis, it helps you design correct gene route with appropriate BioBricks.</p></div>
                </div>
                  <div class="underline-tiny"></div>

          
                  <h5>4. Data Sharing</h5>
                  <p>Hence, while offering various functions, Biopano also serves as a cooperative network building platform. When you create a project, you can add new nodes, name new biological parts and link them with respect to your comprehension to build a brand new net gradually. Meanwhile, Biopano provides log in service. You can sign up with Google or Baidu account and upload your network to the main database to make it more plentiful. Biopano also supports data importing in batch. You can even import data of a species and a database.</p>
       <ul class="small-block-grid-2">
          <li><img src="http://2014.igem.org/wiki/images/2/2a/2014ustc-Feature-f.png"></li>
          <li><img src="http://2014.igem.org/wiki/images/5/57/2014ustc-Feature-g.png"></li>
          </ul>
          <div class="underline-tiny"></div>
       
                  <h5>5. Details and Reference</h5>
                  <p>When you have already got the entire view and want to know more about the details, double-click the nodes and arcs and the details will be shown. More surprisingly, Biopano supports literature reading in the network. You can view literature corresponding to every node and arc along any gene route in the network, and you will understand how the nodes in the net are associated with each other in depth. If you cast doubt on the reliability of our database, you can verify your thought with authoritative literature.</p>
     
        </ul>
      <h4 id="features">Demo</h4>
      <div class="underline-small">
      </div>
      <div>
        <h4>Next we will demonstrate some common modules of E.coli k-12:</h4>
     
          <h5>1, Lac Operon: When you input LacZYA, a Transcription Unit(TU) of Lac Operon and Expand the node:</h5>
          <p>You will see four promoters, acZp1, lacZp2, lacZp3 and lacZp4 as
upstream sequences and three structure genes, lacZ, lacY and lacA as 
downstream sequences. Meanwhile, you can see the TU belongs to 
the Operon lacZYA.
         </p>
          <p>Above the node lacZYA, you also see three Transcription
Factors, namely, CRP, H-NS and lacl. They suppress the expression of 
lazZYA.</p>
          <p>Expand lacZ, lacY and lacA, and we get BgaL, LacY and ThgA
respectively, of which Bgal and ThgA are Enzymes. They catalyze
corresponding reactions. Expand Lacl and we get the gene lacl which
codes it.</p>
       <ul class="small-block-grid-2">
          <li><img src="http://2014.igem.org/wiki/images/4/47/2014ustc-B.png"></li>
          <li><img src="http://2014.igem.org/wiki/images/e/ee/C.png"></li>
          </ul>
        
          <p>Continue to Expand lacZYA and lacl and we find more materials’
connected with them. You can select and drag the nodes to arrange the 
network neatly; you can also click Full Screen and the network will be 
displayed in full screen.</p>
          <ul class="small-block-grid-2">
          <li><img src="http://2014.igem.org/wiki/images/c/c5/2014ustc-D.png"></li>
          <li><img src="http://2014.igem.org/wiki/images/3/3a/2014ustc-E.png"></li>
          </ul>
       
          <h5>2, Following the same steps, we can also get the regulation network of
the Operon Trp.</h5>
          <img src="http://2014.igem.org/wiki/images/9/9d/2014ustc-F.png">
       

      </div>


         <h4>Why choose BioPano?</h4>
      <div class="underline-small"></div>
      <div>
       <p> Biopano ,do the experiment as you do in the lab.</p>
       <p>In experiment, synthetic biologists always find gene circuits formed by BioBricks are incompatible with the host cell, thus cannot work as expected. However, Biopano uses sequence alignment to analyze the possible reasons of the failure and design more acute gene circuit for your consideration.</p>
       <p>Our Blast algorism derives from NCBI. However, the original version only offers sequence alignment and fails to analyze what impact the host environment might have on the injected gene circuit, let along predicts whether it will work.</p>
       <p>Firstly, Biopano offers BioBrick searching to make it convenient for you to study all standardized biological parts on the official website of iGEM. Secondly, you can input a BioBrick and even construct a brand new sequence. Biopano will find the gene nodes of E.coli K-12 sharing the deepest homology with it based on authoritative Blast algorithm. Then you can Expand these nodes to find out the possible regulation on them. In this way, Biopano helps you analyze whether the injected gene circuit will actually work.</p>
       <p>Biopano abstracts biological parts and the regulation among them into nodes and arcs to realize data visualization, and displays gene regulation and metabolism network of E.coli K-12 dynamically by continually Expanding nodes. Single database cannot make an entire material network, so our back end combines Regulon DB, KEGG, Uniprot and abundant data. You can ceaselessly Expand the nodes in the network to comb the relationship among them and satisfy your curiosity.</p>
       
       <p>Compared with Biopano, Cytoscape and Netscience mainly support file importing and network displaying without powerful database from the back end. Hence you cannot search for biological parts online and view the certain part of the biological network you need with them.</p>
       
       <p>Now that there is enormous astonishing material relationship in biological body, more and more biologists hope to find the specific function route between two biological parts or even two biological blocks that seem not related to each other, such as c-di-GMP signalling and Quorum Sensing. Unfortunately, the interactions of the biological parts are saved in database as records and it is almost impossible to find the route in thousands of records. There haven’t been any software to offer any help before. Luckily, Biopano brings hope to this.</p>
      
       <p>You can input two biological parts and set several shortest routes, then you will find their relationship. We used this powerful function to find the specific relationship betweem c-di-GMP signalling and Quorum Sensing, which was corroborated by corresponding literature. How amazing!</p>
       
       <p>We have to admit that Biopano may show you wrong gene route. However, it provides you with direction of studying the relationship of two fields. It not only makes your goal for experiment designing clearer, but enables you to understand the connection inside body is so close as well.</p>
       
       <p>While offering function services, Biopano serves as a platform for collaborative network building. You can define new nodes and arcs yourself and sign in with your Google or Baidu account to upload the network you build to our database. You can also invite several companions to complete a project together. Common biological software, such as Cytoscape and Netscience, doesn’t provide a good platform for communication, while Biopano enhances the communication of biological areas by data sharing.</p>
      
       <p>Biopano, your right hand for biological research. Get it now!</p>
      </div>

       <h4>Future Work</h4>
      <div class="underline-small"></div>
      <div>
       <h4>It is undeniable that we still have a lot to do to improve Biopano in many aspects in order to make it even better. We need to think more and develop it as we can. In the future, we hope to:
</h4>
       
        <p>1. Integrate more data in it. Currently, Biopano only integrates data of E.coli k-12 from Regulon DB, KEGG and Uniprot, and our data in regulatory interaction lack accurate numerical data to indicate its strength. etc. Therefore, not only will we update the existing data in the original species, we will also make complement of high-throughput data through other databases and papers, as well as expand information of other species, so that more biological researchers can benefit from our software.</p>
     
        <p>2. Normalize users' rights, improve quality of data. With the increase of the number of users, the data synchronized every day become not only massive, but also in many different forms, and some of them have nevitable errors or redundant information, or there may be different kinds of comprehension of certain regulatory relations. So we want the machine to automatically recognize the validity of the data submitted by the user, and establish a evaluation criteria of credibility. Thus, only those reliable users will be selected, and only their data can be synchronized to the main database, and the redundancy or errors will be eliminated, and they also continue to enhance the credibility and value of the main database.</p>
    
        <p>3. Enhance network analysis functions. When we add the high-throughput kinetic data for biological networks, we can build a variety of mathematical models on the network and carry out quantitative simulation analysis. For example, we can put the nodes that are closely related to one another together in a group. We can make out software automatically arrange the modules for different functions in the organism. We can even do things that are analogic to those in an electric circult. The network provides the entrance and the exit of the signals, and all you need to do is to enter the target product and we will help you find the substance needed for synthesizing it. </p>
   
        <p>4. Supports more file formats. Currently, the file formats we provide to the users are only XML and SBOL. In the future we will make our software support more file formats, such as sif, EMBL, etc., and we will develop its ability to be capable with other biological softwares such as cytoscape and Netscience, seamlessly. You can import and export any file that are constantly used. From this aspect, we will greatly broadens the scope of the functions of the software.</p>
      
      </div>
     </div>

  </div>
</div>

<div id="last-info" class="hide-for-small">
  <div class="row fullWidth">
    <div class="large-3 medium-3 columns " style="margin-top:5em">
      <img src="http://2014.igem.org/wiki/images/7/78/2014ustc-software2.png">
    </div>
    <div class="large-6 medium-6 columns" style="color:#fff">
      <h5 style="color:#fff;font-weight:bold;margin-bottom: 25px;">Innovation Experiment Centre of Life Science (IEC)</h5>
      <p>Building of Life Sciences, Room 363</p>
      <p>University of Science and Technology of China</p>
      <p>96, Jinzhai Rd, Shushan, Hefei</p>
      <p>230026, Anhui, China</p>
      <p>ustc-software-2014@googlegroups.com </p>
    </div>
   <div class="large-3 medium-3 columns " style="margin-top:5em">
      
      <a href="https://www.db.com" target="_blank"><img src="http://2014.igem.org/wiki/images/c/c0/2014ustc-Dbank.png" style="margin-top:2em"></a>
    </div>

  
  </div>
</div>

<div class="ltx-footer-bottom">
<div class="row" >
  <div class="medium-4 medium-4 push-9 columns">
    <ul class="home-social">
    <li><a href="http://www.twitter.com/USTC_Software" class="twitter"></a></li>
    <li><a href="http://www.facebook.com/USTCSoftware" class="facebook"></a></li>
    <li><a href="#contactus" class="mail"></a></li>
    </ul>
  </div>
  <div class="medium-8 medium-8 pull-5 columns">
    <a href="http://en.ustc.edu.cn"><h4 style="color:antiquewhite">University of Science and Technology of China</h4></a>
    <p class="copyright">© 2014 USTC-Software,University of Science and Technology of China. All rights reserved.</p>
  </div>
</div>
</div>
     <script src="http://2014.igem.org/Team:USTC-Software/jquery.js?action=raw&ctype=text/javascript"></script>
<script>
$(function(){
    $('.row .nameplate').click(function(){
        var description = $(this).parent().find('.details').html();
    var titleText = $(this).text();//text();
    $('#pop_up_content').html('<h2>' + titleText + '</h2>' + description);
        $('#pop_up').fadeIn();
        return false;
    });
    $('.row .profilewrap').click(function(){
        var description = $(this).find('.details').html();
    var titleText = $(this).find('.nameplate').text();//text();
    $('#pop_up_content').html('<h2>' + titleText + '</h2>' + description);
        $('#pop_up').fadeIn();
        return false;
    });    
    $('#pop_up #pop_up_close').click(function(){
        $('#pop_up').fadeOut();
    });
    $('#pop_up #pop_up_underlay').click(function(){
        $('#pop_up').fadeOut();
    });
  /*
    $(".profilewrap").hover(function(){
    $(this).find('.normal').hide();
    $(this).find('.crazy').show();
    },function(){
    $(this).find('.normal').show();
    $(this).find('.crazy').hide();
    }); 
*/
});
</script>
      <script src="http://2014.igem.org/Team:USTC-Software/main.js?action=raw&ctype=text/javascript"></script> <!-- Resource jQuery -->
     
    <script src="http://2014.igem.org/Team:USTC-Software/foundation.min.js?action=raw&ctype=text/javascript"></script>
    <script>
      $(document).foundation();
    </script>
    
  </body>
</html>